README.md

```diff
@@ -129,6 +129,23 @@ The following variables are required:
 - `DAPP_DID_URI` = _This is the URI of the Kilt DID that identifies your dApp_
 - `DAPP_NAME` = _This should be a custom name for your dApp_
 - `JWT_SIGNER_SECRET` = _This is secret key (string) that signs the Json-Web-Tokens before saving them in the Cookies_
+- `CTYPE_HASH` = _This is type of credential (CType) your dApp will request from users for login_
+- `TRUSTED_ATTESTERS` = _This is the DID that identifies the attestors whose attestations are accepted_
+- `REQUIRED_PROPERTIES` = _These are the required properties that identify the CType_
+
+To proceed, you may select the desired credential through the CType Environment Variables listed below, allowing you to tailor the authentication process to your needs.
+
+| Socail KYC | CTYPE_HASH                                                         | TRUSTED_ATTESTERS(spiritnet)                              | TRUSTED_ATTESTERS(peregrine)                              | REQUIRED_PROPERTIES |
```

Comment:
The whole table just has some useful information about some recommended cTypes. I would give it an according description and also a **title** (via `###`on markdown). Maybe "Info about example/recommended cTypes" or something like that. 

Please, also signalise somehow that we only recommend SocialKYC and the types of credentials it issues. 
This is also a good place to explain why the cType-hash is the same for both chains.

The first row is not great. The first column should be "Title", not "SocialKYC". 

Both `TRUSTED_ATTESTERS(spiritnet)` and `TRUSTED_ATTESTERS(peregrine)` should rather be `SocialKYC on Spiritnet` or on `SocialKYC on Peregrine`. 

Maybe you can first list the two SocialKYC DIDs and then add references to them on the table. Then you can leave the title Trusted Attesters. That would be more fitting, if later we decide to add an external attester to this list.

You will have to make them subtitles to be able to reference them, like this: 
```
####SocialKYC on the Production: did:kilt:...

And then inside the table (prod. SYKC)[#SocialKYC on the Production]
```

And on the properties column is maybe better to include all properties and let them decide. This would make the "required" part false. But the columns name really don't need to be the name of the CONSTANTS.

Reply:
Done at: [4fe1b4e](https://github.com/KILTprotocol/web3-login-demo/pull/78/commits/4fe1b4e7e582b11bcf63229d6331fab8e0c7a871)